[Map] Fix 2.23/2.24 changelog

zak39 opened the pull request:
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no <!-- please update src/**/CHANGELOG.md files -->
| Docs?         | yes <!-- required for new features -->
| Issues        | Chore  <!-- prefix each issue number with "Fix #", no need to create an issue if none exist, explain below instead -->
| License       | MIT

<!--
Replace this notice by a description of your feature/bugfix.
This will help reviewers and should be a good start for the documentation.

Additionally (see https://symfony.com/releases):
 - Always add tests and ensure they pass.
 - For new features, provide some code snippets to help understand usage.
 - Features and deprecations must be submitted against branch main.
 - Update/add documentation as required (we can help!)
 - Changelog entry should follow https://symfony.com/doc/current/contributing/code/conventions.html#writing-a-changelog-entry
 - Never break backward compatibility (see https://symfony.com/bc).
-->

The following features were removed from the changelog as they are not present in the 2.23 release:

-  `id` to `Marker`, `Polygon` and `Polyline` constructors
-  `Map::removeMarker(string|Marker $markerOrId)`
-  `Map::removePolygon(string|Polygon $polygonOrId)`
-  `Map::removePolyline(string|Polyline $polylineOrId)`

Kocal reviewed (changes requested):
The lines you removed must be moved to 2.24, as the features are still present in the repository

zak39 committed:
chore: Adjust changelog for features moving to 2.24 release

The following features were moved in the changelog as they are not present in the 2.23 release, but will be included in the 2.24 release

-  `id` to `Marker`, `Polygon` and `Polyline` constructors
-  `Map::removeMarker(string|Marker $markerOrId)`
-  `Map::removePolygon(string|Polygon $polygonOrId)`
-  `Map::removePolyline(string|Polyline $polylineOrId)`

zak39 commented:
I fixed it, I apologize for my mistake 🙏

Kocal commented:
<details>
<summary>Don't worry, mistake happens ✨ </summary>

![image](https://github.com/user-attachments/assets/ba069965-dfc8-4419-941f-ccfa6973a914)


</details>

Kocal commented:
Thank you @zak39.

zak39 commented:
> Thank you @zak39.

You're welcome ! :wink:
